__tests__/app.test.js

```diff
@@ -127,6 +127,191 @@ describe("GET /api/articles", () => {
   });
 });
 
+describe("GET /api/articles?sort_by", () => {
+  test("200: responds with articles sorted by created_at in descending order by default when given no sort_by query", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles.length).toBe(13);
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+      });
```

Comment:
If you previously tested the above then you don't need it again. If you didn't ignore me 😄